content/references/protocol-reference/transactions/transaction-types/clawback.md

```diff
@@ -0,0 +1,64 @@
+---
+html: clawback.html
+parent: transaction-types.html
+blurb: Claw back an issued currency transfer between accounts in a trust set.
+labels:
+  - Tokens
+status: not_enabled
+---
+# Clawback
+
+[[Source]](https://github.com/XRPLF/rippled/blob/master/src/ripple/app/tx/impl/Clawback.cpp "Source")
+
+<embed src="/snippets/_clawback-disclaimer.md" />
+
+Claw back an issued currency transfer between accounts in a trust set.
+
+## Example Clawback JSON
+
+```json
+{
+  "TransactionType": "Clawback",
+  "Account": "rp6abvbTbjoce8ZDJkT6snvxTZSYMBCC9S",
+  "Amount": {
+      "currency": "FOO",
+      "issuer": "rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW",
+      "value": "314.159"
+    }
+}
+```
+
+[Clawback example transaction. >](websocket-api-tool.html?server=wss%3A%2F%2Fs.devnet.rippletest.net%2F&tx=%7B%22id%22%3A%22example_Clawback%22%2C%22command%22%3A%22Clawback%22%2C%22Account%22%3A%22rp6abvbTbjoce8ZDJkT6snvxTZSYMBCC9S%22%2C%22Amount%22%3A%7B%22currency%22%3A%22FOO%22%2C%22issuer%22%3A%22rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW%22%2C%22value%22%3A%22314.159%22%7D%7D)
+
+## Clawback Fields
+
+In addition to the common fields, `Clawback` transactions use the following fields:
+
+
+| Field              | JSON Type | [Internal Type][] | Description       |
+|:-------------------|:----------|:------------------|:------------------|
+| `TransactionType`  | string    | `UINT16`          | Indicates the new transaction type `Clawback`. The integer value is `30`. The name is `ttCLAWBACK`. |
+| `Account`          | string    | `ACCOUNT ID`      | The account executing this transaction. The account must be the issuer of the asset being clawed back. Note that in the XRP Ledger, trust lines are bidirectional and, under some configurations, both sides can be seen as the "issuer" of an asset. In this specification, the term *issuer* is used to mean the side of the trust line that has an outstanding balance (that is, 'owes' the issued asset) that it wants to claw back.|
+| `Flags`            | number    | `UINT32`          | (Optional) The universal transaction flags that are applicable to all transactions (for example, `tfFullyCanonicalSig`) are valid. This proposal introduces no new transaction-specific flags. |
```

Comment:
This is another common field and should not be listed here.

Reply:
Thank you, Rome, for your thorough review.